src/storage/src/hummock/utils.rs

```diff
@@ -75,7 +76,8 @@ pub fn validate_table_key_range(levels: &[Level]) -> HummockResult<()> {
     Ok(())
 }
 
-pub fn filter_single_sst<R, B>(info: &SstableInfo, key_range: &R) -> bool
+/// `table_id == 0` represents that no `table_id` filter
```

Comment:
I think we will have table_id in all storage read interface. Therefore we can always get the table id before we call this method, and then we can avoid this `table_id == 0 represents that no `table_id` filter` hack. If this is only for test, we can enable this hack only in test.

Reply:
> I think we will have table_id in all storage read interface. Therefore we can always get the table id before we call this method, and then we can avoid this `table_id == 0 represents that no `table_id` filter` hack. If this is only for test, we can enable this hack only in test.

fixed